docs/howto/delete-app.txt

```diff
@@ -0,0 +1,22 @@
+==================================
+How to delete a Django application
+==================================
+
+Django provides the ability to group sets of features into Python packages
+called :doc:`applications</ref/applications/>`. However, when requirements
+change, apps may become obsolete or unnecessary. The following guide will help
+you delete an application safely.
+
+1. Remove all references to the app (i.e imports, foreign keys).
+
+2. Remove all models from your ``models.py`` file.
+
+3. Generate your migrations running :djadmin:`makemigrations`.
```

Comment:
```suggestion
3. Create relevant migrations by running :djadmin:`makemigrations` (this step will generate a migration that removes all tables for the removed models).
```

Reply:
Applied! I extended it a bit and wrote `[..] and any other migration in case there was a relationship` in case step 1 there were foreign keys deleted and generate other migrations